src/main/g8/src/main/scala/$organization__packaged$/Spark.scala

```diff
@@ -0,0 +1,18 @@
+package $organization$
+
+import org.apache.spark.{SparkConf, SparkContext}
+import org.apache.spark.sql.SparkSession
+
+import scala.util.Properties
+
+object Spark {
+  def conf: SparkConf = new SparkConf()
+    .setIfMissing("spark.master", "local[*]")
+    .setAppName("$name$")
+    .set("spark.serializer", "org.apache.spark.serializer.KryoSerializer")
+    .set("spark.kryo.registrator", "geotrellis.spark.store.kryo.KryoRegistrator")
+    .set("spark.executionEnv.AWS_PROFILE", Properties.envOrElse("AWS_PROFILE", "default"))
+
+  implicit val session: SparkSession = SparkSession.builder.config(conf).enableHiveSupport.getOrCreate
+  implicit def context: SparkContext = session.sparkContext
```

Comment:
Hm, what is the logic here to define defs and vals? 
Mb it is safe to make them `@transient lazy val`?